synth.py

```diff
@@ -17,16 +17,14 @@
 import synthtool as s
 from synthtool import gcp
 
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICBazel()
 common = gcp.CommonTemplates()
 version = 'v2'
 
 library = gapic.py_library(
-    'bigquery',
-    version,
-    config_path='/google/cloud/bigquery/'
-                'artman_bigquery_v2.yaml',
-    artman_output_name='bigquery-v2',
+    service='bigquery',
+    version=version,
+    bazel_target=f"//google/cloud/bigquery/{version}:bigquery-{version}-py",
```

Comment:
The library still needs to support Python 2.7 and 3.5, thus f-strings cannot be used, unfortunately.

Reply:
Synth.py just calls bazel, and for what bazel is doing, it is ok with any version of python. And yes, synth.py is not what end users call, it is used just to generate the library itself.